src/main/java/org/eolang/EOarray.java

```diff
@@ -0,0 +1,103 @@
+package org.eolang;
+
+import org.eolang.core.EOObject;
+import org.eolang.core.EOObjectArray;
+import org.eolang.core.data.EOData;
+import org.eolang.core.data.EODataObject;
+
+public class EOarray extends EOObject {
+    private EOObjectArray array;
+
+    public EOarray(EOObjectArray array){
```

Comment:
Actually, it would be great to remove EOObjectArray and make EOarray as the only class working with arrays. 
I will describe it in detail later.